### PR TITLE
Updating EDA controller user guide asciidoc

### PR DESCRIPTION
**Pull request by EMcWhinn**
Making necessary changes as per content audit

Updating EDA controller user guide asciidoc elements

https://issues.redhat.com/browse/AAP-13856

Affects `titles/eda-user-guide`

### CODE REVIEW

File: downstream/assemblies/eda/assembly-eda-projects.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[id="eda-projects"]
+
+= Projects
+
+[role="_abstract"]
+
+Projects are a logical collection of rulebooks.
+
+include::eda/proc-eda-set-up-new-project.adoc[leveloffset=-0]
```
Comment (dcdacosta) [inline downstream/assemblies/eda/assembly-eda-projects.adoc:9]: This should be [leveloffset=+1] which indicates that it is a subsection of this assembly. Then in the procedure module, the title should have only 1 = preceding the title text. I'll comment there as well. I believe nesting was previously managed with the = in the modules but now that we're using assemblies, the nesting is controlled by the leveloffset.

File: downstream/modules/eda/proc-eda-set-up-new-project.adoc [hunk 2]
```diff
@@ -1,20 +1,20 @@
 [id="eda-set-up-new-project"]
 
-= Setting up a new project
+== Setting up a new project
```
Comment (dcdacosta) [inline downstream/modules/eda/proc-eda-set-up-new-project.adoc:3]: This should be = Setting up a new project and the extra = should be removed. See above for the explanation and don't hesitate to let me know if you need me to clarify further.

**Review by dcdacosta — APPROVED** [review]
Hi @EMcWhinn, Great job on this, I left just a few minor comments.

File: downstream/assemblies/eda/assembly-eda-projects.adoc (hunk 1 above)
Comment (EMcWhinn) [reply downstream/assemblies/eda/assembly-eda-projects.adoc:9]: Done :)

**Review by dcdacosta — APPROVED** [review]
LGTM!

**Pull request merged**